howto/monitor/landing.md

```diff
@@ -1,10 +1,32 @@
 (howto-monitor-anbox)=
 # How to monitor Anbox Cloud
 
-Anbox Cloud gathers various performance metrics and makes them accessible through API endpoints. While Anbox Cloud does not provide its own observability solution, it supports implementing and integrating custom solutions for monitoring performance.
+Anbox Cloud collects various metrics and makes them accessible through API endpoints. While Anbox Cloud does not provide its own observability solution, it supports integrating with external solutions.
 
-See {ref}`ref-prometheus-metrics` for a list of available metrics and how to access them. [LXD Metric exporter for instances](https://discuss.linuxcontainers.org/t/lxd-metric-exporter-for-instances/11735) has a list of metrics available through LXD.
+You can find a list of available metrics at {ref}`ref-prometheus-metrics`.
 
-The implementation of a monitoring or observability solution depends on your specific use case and the tools that you want to use. See [Open source observability for enterprises](https://ubuntu.com/observability) for an overview of observability tools and stacks that are recommended and supported by Canonical. In particular, see the [Canonical Observability Stack (LMA2)](https://juju.is/docs/lma2).
+## Access metrics with the appliance
 
-There are plans in the Anbox Cloud roadmap to provide better monitoring abilities by using the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack).
+The Anbox Cloud Appliance provides a central metrics endpoint which aggregates metrics from all internal services and Anbox instances.
+
+If you haven't enabled metrics support at initialization time, you can do that now by running
+
+    sudo anbox-cloud-appliance enable metrics
+
+If you want to disable metrics again you can run
+
+    sudo anbox-cloud-appliance disable metrics
+
+Afterwards you can find the URL which you can use to retrieve the metrics in the [Prometheus data format](https://prometheus.io/docs/concepts/data_model/).
+
+    sudo anbox-cloud-appliance config show
+
+The output will list a `metrics.url` field which provides you the URL. The output will also provide a TLS certificate the endpoint the URL points to is using to establish a secure and authenticated connection.
+
+You can either configure a Prometheus instance to scrape the endpoint or manually retrieve the metrics via `curl`, for example
+
+  metrics_url="$(sudo anbox-cloud-appliance config show | yq .metrics.url)"
+  sudo anbox-cloud-appliance config show | yq .metrics.tls.certificate > metrics.pem
```

Comment:
yq is not that well-known compared to jq, so I'm not sure if that will not bring questions from users.

Reply:
You're right, let me add a hint about `yq`.